{
  "stats": {
    "suites": 2,
    "tests": 2,
    "passes": 1,
    "pending": 0,
    "failures": 1,
    "start": "2023-10-11T13:08:36.336Z",
    "end": "2023-10-11T13:08:49.635Z",
    "duration": 13299,
    "testsRegistered": 2,
    "passPercent": 50,
    "pendingPercent": 0,
    "other": 0,
    "hasOther": false,
    "skipped": 0,
    "hasSkipped": false
  },
  "results": [
    {
      "uuid": "89d1872d-4b60-4a62-97a3-6324796f12e8",
      "title": "",
      "fullFile": "cypress\\e2e\\AutoTests\\Customer\\Login.cy.js",
      "file": "cypress\\e2e\\AutoTests\\Customer\\Login.cy.js",
      "beforeHooks": [],
      "afterHooks": [],
      "tests": [],
      "suites": [
        {
          "uuid": "ebee36b5-69d1-4b4b-b8a0-561ae641223d",
          "title": "Login CustomerPanel",
          "fullFile": "",
          "file": "",
          "beforeHooks": [],
          "afterHooks": [],
          "tests": [
            {
              "title": "Login Successfully",
              "fullTitle": "Login CustomerPanel Login Successfully",
              "timedOut": null,
              "duration": 5939,
              "state": "passed",
              "speed": "medium",
              "pass": true,
              "fail": false,
              "pending": false,
              "context": null,
              "code": "cy.visit(`${_urls.URL}${_urls.customer}`);\ncy.wait(2000);\nlet login = new _home.Login();\nlogin.usernameInput();\nlogin.passwordInput();\ncy.get('[class=\"q-page q-layout-padding bg-white q-pt-lg\"]').should('be.visible');",
              "err": {},
              "uuid": "eaeeb307-5bf1-44ec-961a-8d73e43ae240",
              "parentUUID": "ebee36b5-69d1-4b4b-b8a0-561ae641223d",
              "isHook": false,
              "skipped": false
            }
          ],
          "suites": [],
          "passes": [
            "eaeeb307-5bf1-44ec-961a-8d73e43ae240"
          ],
          "failures": [],
          "pending": [],
          "skipped": [],
          "duration": 5939,
          "root": false,
          "rootEmpty": false,
          "_timeout": 2000
        },
        {
          "uuid": "7a341915-81f5-4bb9-bad6-a607450bff36",
          "title": "Logout CustomerPanel",
          "fullFile": "",
          "file": "",
          "beforeHooks": [],
          "afterHooks": [],
          "tests": [
            {
              "title": "Logout Successfully",
              "fullTitle": "Logout CustomerPanel Logout Successfully",
              "timedOut": null,
              "duration": 4202,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.visit(`${_urls.URL}${_urls.customer}`);\nlet Login = (0, _login.loginCustomer)();\ncy.request({\n  method: 'POST',\n  url: `${Login.baseUrl.adminBaseURL()}/login`,\n  body: Login.body\n}).as('login-user');\ncy.get('@login-user').then(res => {\n  let account = JSON.stringify(res.body);\n  window.localStorage.setItem('ACCOUNT', account);\n});\ncy.reload();\ncy.wait(2000);\ncy.gclick('[href=\"/dashboard\"]');\ncy.wait(500);\ncy.get('div').contains('خروج').click();\ncy.wait(500);\ncy.get('.full-height').should('be.visible');",
              "err": {
                "message": "CypressError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > \n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application",
                "estack": "CypressError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > \n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\nError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > \n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\nError",
                "diff": null
              },
              "uuid": "462999af-d107-4f62-8b63-549fe4005285",
              "parentUUID": "7a341915-81f5-4bb9-bad6-a607450bff36",
              "isHook": false,
              "skipped": false
            }
          ],
          "suites": [],
          "passes": [],
          "failures": [
            "462999af-d107-4f62-8b63-549fe4005285"
          ],
          "pending": [],
          "skipped": [],
          "duration": 4202,
          "root": false,
          "rootEmpty": false,
          "_timeout": 2000
        }
      ],
      "passes": [],
      "failures": [],
      "pending": [],
      "skipped": [],
      "duration": 0,
      "root": true,
      "rootEmpty": true,
      "_timeout": 2000
    }
  ],
  "meta": {
    "mocha": {
      "version": "7.0.1"
    },
    "mochawesome": {
      "options": {
        "quiet": false,
        "reportFilename": "mochawesome",
        "saveHtml": false,
        "saveJson": true,
        "consoleReporter": "spec",
        "useInlineDiffs": false,
        "code": true
      },
      "version": "7.1.3"
    },
    "marge": {
      "options": {
        "id": "default",
        "cypressParallel": "true",
        "overwrite": false,
        "html": false,
        "json": true,
        "reportDir": "cypress\\reports\\html\\.jsons"
      },
      "version": "6.2.0"
    }
  }
}